packages/ckeditor5-engine/src/view/renderer.ts

```diff
@@ -562,6 +565,12 @@ export default class Renderer extends Observable {
 			return false;
 		}
 
+		// Do not use inline filler while typing outside inline elements on Android.
+		// The deleteContentBackward would remove part of the inline filler instead of removing last letter in a link.
+		if ( env.isAndroid && ( nodeBefore || nodeAfter ) ) {
```

Comment:
I like the comment. Could we also add a link to a ticket? I think we had this reported.

Reply:
Can't find the ticket. I'm not sure if it was written on any ticket.